src/stories/Quiz/Component/QuestionTypes/CoverQuestion.stories.tsx

```diff
@@ -3,27 +3,27 @@ import type { Meta, StoryObj } from '@storybook/react';
 import { CoverQuestion } from '@constructor-io/constructorio-client-javascript/lib/types';
 import CoverTypeQuestion from '../../../../components/CoverTypeQuestion/CoverTypeQuestion';
 import { getMockQuestionWithImage, getMockQuestion } from '../../tests/mocks';
-import QuestionTypeDecorator_v2 from './QuestionTypeDecorator_v2';
+import QuestionTypeDecorator from './QuestionTypeDecorator';
```

Comment:
Migrate all questions to QuestionTypeDecorator_v2 and rename it to QuestionTypeDecorator_v2